src/routes/blog/spyder-6-remote-development/+page.md

```diff
@@ -0,0 +1,94 @@
+It also features graphical status checking and reporting, host configuration management, and more.
+In addition, it brings a new `Consoles` submenu allowing you to connect to any remote environment with one click, launch a console and run your code.
+You can also stop code execution as well as restart remove kernels from right within Spyder.
+
+<video controls>
+  <source src="remote-console-demo.mp4" type="video/mp4">
+  <track kind="captions">
+  Sorry, your browser doesn’t support HTML5 video. Download the <a href="screencast.mp4">MP4 file</a>
+</video>
+
+We're excited to have our newest core developer, Hendrik Louzada, and our team share with you their insights and perspectives on Spyder's new remote development platform introduced in Spyder 6!
+Join us as Hendrik shares how he got started with Spyder and the project, how the new achetecture is implemented under the hood, what challenges we faced and what he learned from them, and what's next for Spyder 6.1.0!
+
+
+## How'd you find out about Spyder and why'd you join the project?
+
+Hendrick first started to work on Spyder around three years ago, developing a series of plugins to control remote magnetic resonance equipment (such as MRI machines) directly in the IDE.
+As part of that project, he made multiple upstream contributions to Spyder itself to improve the Language Server Protocol support (LSP, the architecture that powers Spyder's code completion, introspection, analysis and formatting).
+
+In 2023, Spyder received a [Chan Zuckerberg Initiative](https://chanzuckerberg.com/) (CZI) [EOSS 5 grant](https://chanzuckerberg.com/eoss/proposals/enhancing-spyder-ide-remote-support-for-scientific-research-in-python/) to implement a new remote development platform and features in Spyder to allow users to develop and run code in remote desktops, servers, clusters and cloud machines.
+After a six-month search by Spyder lead maintainer Carlos Cordoba to find the right candidate to engineer the backend and network architecture for this project, he found Hendrik, who joined the core development team near the end of 2023.
+
+
+![New remote connection manager dialog in Spyder listing the configurable settings for a new remote host](remote-connection-manager-new.png)
+
+## How is the new remote development platform implemented?
+
+The backend, [`spyder-remote-services`](https://github.com/spyder-ide/spyder-remote-services), uses [Jupyter Server](https://jupyter-server.readthedocs.io/) and the [JupyterHub API](https://jupyterhub.readthedocs.io/en/stable/reference/rest-api.html), since Spyder is already leverages the Jupyter architecture to manage kernels and run code.
+We decided to use SSH to communicate with the remote machine, since it's secure and widely used for this purpose.
+
```

Comment:
I can definitely swap the image if it makes more sense, but the direct URLs aren't working for me cause they are too old or due to some permissions issue (as it didn't work for me before when I went to the PRs); either let me know on what comment the image is in and what image is it so I can download it from there, or download it and post it again here or elsewhere.

And see above comment why the current position isn't an issue while moving it would be.

Reply:
Just FYI, some time ago GitHub added the feature to comment directly on a file for things that relate to the file itself rather than e.g. its position in a document (so its easier to find and it e.g. sticks around until it is changed or resolved, rather than disappears if the position or injection changes).

Reply:
![connection](https://github.com/user-attachments/assets/194adf1f-e7be-4b71-b30d-4fe8c89b6f49)

Reply:
(Just FYI, since GitHub trims whitespace before and after the changed line(s), so the suggestion region would need to include the prior and following non-whitespace lines to apply correctly)

Reply:
Screenshot updated

Reply:
Agreed with @ccordoba12 to add line breaks and a caption to images instead